virttest/utils_net.py

```diff
@@ -1133,7 +1133,9 @@ def del_port(self, brname, ifname):
         try:
             self._br_ioctl(arch.SIOCBRDELIF, brname, ifname)
         except IOError as details:
-            raise BRDelIfError(ifname, brname, details)
+            # Avoid failing the test when port not present in br
+            if ifname not in self.list_iface(brname):
```

Comment:
If the port has been removed, should this line be `in` instead of `not in`?

Reply:
How embarrassing, of course it should. I'm sorry for hurrying this really simple fix.